### PR TITLE
[Localization] Add localization pokemon fainted (Addition of #1276)

### PR DESCRIPTION
**Pull request by Ohmry**
## What are the changes?

Add localization for pokemon fainted. It is addition of #1276 

## Why am I doing these changes?

It needs to translate of pokemon fainted

## What did change?

- Change queueMessage to i18next.t("battle:fainted") from just "fainted" in phase.ts

#### English
image

#### Korean
image

#### Other languages (in example Deutsch)
image

## How to test the changes?

- Checkout this PR
- Change language

## Checklist
- [X] There is no overlap with another PR?
- [ ] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/battle.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ export const battle: SimpleTranslationEntries = {
   "notDisabled": "{{pokemonName}}'s {{moveName}} ist\nnicht mehr deaktiviert!",
   "eggHatching": "Oh?",
   "ivScannerUseQuestion": "IV-Scanner auf {{pokemonName}} benutzen?",
+  "wildPokemonWithAffix": "{{pokemonName}} (wild)",
+  "foePokemonWithAffix": "{{pokemonName}} (Gegner)",
+  "useMove": "{{pokemonNameWithAffix}} setzt {{moveName}} ein!",
   "drainMessage": "{{pokemonName}} wurde Energie abgesaugt",
-  "regainHealth": "KP von {{pokemonName}} wurden wieder aufgefrischt!"
+  "regainHealth": "KP von {{pokemonName}} wurden wieder aufgefrischt!",
+  "fainted": "{{pokemonNameWithAffix}} kampfunfähig!"
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:64]: When is this used? When the pokemon gets fainted or when it is shown that it is fainted?

Comment (Ohmry) [reply src/locales/de/battle.ts:64]: It is used when pokemon that have lost all HP and fainted in battle.

Comment (CodeTappert) [reply src/locales/de/battle.ts:64]: Then it is "wurde besiegt."

Comment (Ohmry) [reply src/locales/de/battle.ts]: Done (7ebdf90)

**Pull request closed without merging**